salt/_modules/metalk8s.py

```diff
@@ -128,25 +128,25 @@ def _get_product_name(info):
     return match.group('name') if match else None
 
 
-def _get_product_info(info):
-    """Extract product information from info
+def _get_archive_info(info):
```

Comment:
I think we could have left this one (and other similar references) as _**product** info_, since we basically extract it from `product.txt`.

- `archive` describes the ISO or directory, which we configure in `BootstrapConfiguration`
- `product` describes the software that is shipped within an `archive`

Reply:
Was discussed on Slack yesterday: do we only rename the visible part or the whole pillar/salt stuff.
Was agreed to rename everything to be consistent so I'll keep it like that.